.cloudbees/workflows/workflow.yaml

```diff
@@ -48,6 +63,5 @@ jobs:
     - name: docker build and publish
       uses: cloudbees-io/kaniko@v1
       with:
-        dockerconfigjson: ${{ secrets.dockerconfigjson }}
         destination: registry.saas-dev.beescloud.com/staging/helm-uninstall-action:${{ cloudbees.scm.branch }}${{ cloudbees.scm.branch == 'main' && ',registry.saas-dev.beescloud.com/staging/helm-uninstall-action:latest' || '' }}
```

Comment:
Since I came into this late and haven't been able to follow the entire conversation, I maybe missing something. The workflow for this repo is still publishing to the Harbor registry but we are referring to the image from an ECR registry here? How is that supposed to work?

Reply:
I’m not familiar with the ‘workflow/pipeline’ works…
The public registry is not a dev registry - it will only have tested/vetted images that SRE will publish
We can not pin versions as we will not over write versions.

I had approved the actiion.yaml change before the other changes came in.

Reply:
Ok, so the promotion of the image to the ECR registry from Harbor will be managed by SRE, which would also mean that the version in this file will also have to be managed by SRE when they promote a new version of the image to ECR.